Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -75,7 +80,38 @@ extension TabTrayV2ViewController: UITableViewDelegate {
         }
 
         let section = TabSection(rawValue: section)
-        headerView.titleLabel.text = section?.description.uppercased()
+        let sectionHeader: String
+        let date: String
+        
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateStyle = .medium
+        dateFormatter.timeStyle = .none
+        dateFormatter.locale = Locale(identifier: Locale.current.identifier)
+        
+        let dateIntervalFormatter = DateIntervalFormatter()
```

Comment:
Suggestion: 
The date logic can likely go into viewmodel and we can grab it from there. This will take away some load from TabTravyV2ViewController.